test/specs/parameter.js

```diff
@@ -0,0 +1,19 @@
+const firstname = 'uyen'
+const lastname = 'nguyen'
+const email = 'adain.48@dikybuyerj.com'
+const invalid_email ='test@email.com'
+const invalid_value = 'acb'
+const password = 'Test@123'
+const blank_value = ''
+var character = ''
```

Comment:
This file's purpose is constant declarations. Don't declare a variable here that could be changed somewhere else.

Reply:
I fixed it